src/coreclr/jit/gentree.cpp

```diff
@@ -19298,6 +19298,18 @@ GenTree* Compiler::gtNewSimdAbsNode(var_types type, GenTree* op1, CorInfoType si
     {
         return gtNewSimdHWIntrinsicNode(type, op1, intrinsic, simdBaseJitType, simdSize);
     }
+    else if ((simdSize == 64) && ((simdBaseType == TYP_INT) || (simdBaseType == TYP_LONG)))
+    {
+        assert(compIsaSupportedDebugOnly(InstructionSet_AVX512F));
+        NamedIntrinsic intrinsic = NI_AVX512F_Abs;
+        return gtNewSimdHWIntrinsicNode(type, op1, intrinsic, simdBaseJitType, simdSize);
+    }
+    else if ((simdSize == 64) && (varTypeIsSmall(simdBaseType)))
+    {
+        assert(compIsaSupportedDebugOnly(InstructionSet_AVX512BW));
+        NamedIntrinsic intrinsic = NI_AVX512BW_Abs;
+        return gtNewSimdHWIntrinsicNode(type, op1, intrinsic, simdBaseJitType, simdSize);
+    }
```

Comment:
This should ideally be handle a bit higher up by putting the `LONG` path under the existing `LONG` check and adding an `else if (simdSize == 64)` under the `simdSize == 32` handling.

Reply:
Will try to do that and push it